schema/deploy/warehouse/target/organism.sql

```diff
@@ -0,0 +1,15 @@
+-- Deploy seattleflu/schema:warehouse/target/organism to pg
+-- requires: warehouse/target
+-- requires: warehouse/organism
+
+begin;
+
+alter table warehouse.target
+    add column organism_id integer references warehouse.organism (organism_id);
```

Comment:
I wasn't planning to because a) that would break upserting of targets during presence/absence ETL where the organism info isn't available and b) it seems like we'll always have some targets which don't map to organisms nicely (controls, for example).

Reply:
Doe that mean each time a new target gets added to the database we would have to link it to an organism manually?

Reply:
Yep. Do you have thoughts on a better approach here? Would definitely be interested to hear any ideas.

We can create targets (with organisms) ahead of presence/absence data if BBI communicates the new diagnostic panels to us in advance, otherwise we can backfill it afterwards.

Reply:
I can't really think of a better approach here. I do think it would be best if we can setup/create targets with organisms ahead of the presence/absence data

Reply:
Agreed!